Yandex/Yandex/MailPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Threading;
+
+namespace Yandex
+{
+    public class MailPage : BasePage
+    {
+        public const string InboxButtonXPath = "//a[@href='#tabs/relevant']";
+        const string ProfileButtonClassName = "user-account";
+        const string SignOutButtonClassName = "legouser__menu-item_action_exit";
+
+        Dictionary<string, IWebElement> _elementCollection;
```

Comment:
Why do you use Dictionary to store web elements?

Reply:
I was told it is the best practice because it helps to read code as a scenario.

Reply:
@alexsavenok-savenduk please review changes.